### PR TITLE
Project list index javascript implementation.

### PR DESCRIPTION
**Pull request by Joeper214**
#### Searchable, Pagination and Sorting implelentation.

1. Added Browserify to modularize the filters and other functionalities needed for searchable list in vue.
2. Implement search, sorting and pagination in project list.
3. Implement search, sorting and pagination in infrastructure list (Modularized).
4. UI wrap up to index files of projects page.

### CODE REVIEW

File: app/assets/javascripts/projects.js [hunk 1]
```diff
@@ -5,4 +5,110 @@
...
+         $("#loading").hide();
+    },
+    filters:{
+      wrap: wrap,
+      listen: listen,
+      paginate: function(list) {
+        var index = this.pageNumber * this.pages;
+        return list.slice(index, index + this.pages);
+      },
+      roundup: function (val) { return (Math.ceil(val))},
```
Comment (skyarch-networks) [inline app/assets/javascripts/projects.js]: __[jshint]__
*  Missing semicolon.

Comment (Joeper214) [inline app/assets/javascripts/projects.js]: Alright! I'll fix this.

**Comment by Joeper214** [comment]
How can we fix this conflicts? @pocke

**Comment by pocke** [comment]
Maybe merge develop into branch of this PR and solve conflicts manually.

**Pull request merged**